<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <class>NA_Qt_test2Class</class>
 <widget class="QMainWindow" name="NA_Qt_test2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NA_Qt_test2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="deviceInfo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>651</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="contentBrowser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>711</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="na_qt_test2.qrc"/>
 </resources>
 <connections/>
</ui>
